spec.html

```diff
@@ -16171,20 +16171,19 @@
           <p>The following is an informative definition of an ECMAScript generator function that conforms to these rules:</p>
           <pre><code class="javascript">
             function* EnumerateObjectProperties(obj) {
-              let visited = new Set;
-              for (let key of Reflect.ownKeys(obj)) {
-                if (typeof key === "string") {
-                  let desc = Reflect.getOwnPropertyDescriptor(obj, key);
-                  if (desc && !visited.has(key)) {
-                    visited.add(key);
-                    if (desc.enumerable) yield key;
-                  }
+              const visited = new Set;
+              for (const key of Reflect.ownKeys(obj)) {
+                if (typeof key === "symbol") continue;
+                const desc = Reflect.getOwnPropertyDescriptor(obj, key);
+                if (desc && !visited.has(key)) {
+                  visited.add(key);
+                  if (desc.enumerable) yield key;
                 }
               }
-              let proto = Reflect.getPrototypeOf(obj)
+              const proto = Reflect.getPrototypeOf(obj);
```

Comment:
Any JavaScript code contains opinionated parts: this function will be just fine without semicolons and strict equals. Does TC39 advise against `==` and ASI too?